README.md

```diff
@@ -2,41 +2,47 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Getting Started
 
-### Initial setup
+### Initial setup and subsequent updates
 
 1. Install nvm if you don't have it yet.
-2. Activate the right node version
+2. Activate the right node version (you can do all steps from 2 to 5 to make sure that the update does not fail).
```

Comment:
The text in brackets isn't very clear without context, I suggest removing that and adding this before the numbered list:

```
> Note: If dependency installation fails on subsequent npm installs, you may have an invalid npm token due to a caching issue on npm.kausal.tech. Follow steps 2 to 5 to fix this.
```